internal/monitors/collectd/solr/solr.go

```diff
@@ -0,0 +1,106 @@
+package solr
+
+import (
+	"github.com/signalfx/signalfx-agent/internal/core/config"
+	"github.com/signalfx/signalfx-agent/internal/monitors"
+	"github.com/signalfx/signalfx-agent/internal/monitors/collectd"
+	"github.com/signalfx/signalfx-agent/internal/monitors/collectd/python"
+	"github.com/signalfx/signalfx-agent/internal/monitors/pyrunner"
+)
+
+const monitorType = "collectd/solr"
+
+// MONITOR(collectd/solr): Monitors Solr instances.
+//
+// See https://github.com/signalfx/collectd-solr and
+// https://github.com/signalfx/integrations/tree/master/collectd-solr
+//
+// Sample YAML configuration:
+//
+// ```yaml
+// monitors:
+// - type: collectd/solr
+//   host: 127.0.0.1
+//   port: 8983
+// ```
+//
+// Sample YAML configuration with list lengths:
```

Comment:
"list lengths"?? Did you copy/paste this from the Redis monitor?

Reply:
Yes